### PR TITLE
Support joining a game

### PR DESCRIPTION
**Pull request by jeffp2007**
Need to update black_uid attribute field to current_user.id once we add authentication. Right now it's set to an arbitrary value.

### CODE REVIEW

File: app/controllers/games_controller.rb [hunk 1]
```diff
@@ -1,2 +1,26 @@
 class GamesController < ApplicationController
+
+	def index
+		@games = Game.all
+		@open_games = Game.where(:black_uid => nil)
```
Comment (johnnyiller) [inline app/controllers/games_controller.rb:5]: This won't capture games that don't have a white_uid.  My guess is that you are going to always assign the first user to the white side of the board.  This works but removes the option for users to choose a color.  Just want to make sure you understand the tradeoff.

**Comment by johnnyiller** [comment]
this looks good.

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (cryogenx) [inline app/controllers/games_controller.rb:5]: generally speaking, the person who initiates the game would be white

Comment (jeffp2007) [inline app/controllers/games_controller.rb:5]: Yes, in the task it said we were assigning a user joining a game to
black_uid, so my assumption was the game creator would have been assigned
white_uid.

## 

Jeff Powers
Global Partners, Inc.
www.globalpartnersinc.com

Connect on Linkedin: http://www.linkedin.com/in/jeffreytpowers/

**Comment by johnnyiller** [comment]
So you should fix any conflicts this branch has with master and then push it back up for one final review.  Then we can merge it in if everything looks good.

File: app/controllers/games_controller.rb (hunk 1 above)
Comment (johnnyiller) [inline app/controllers/games_controller.rb:5]: Yes that's the correct assumption, I just wanted to make sure it was intentional, which it clearly was.  Good work.

**Comment by jeffp2007** [comment]
Thanks. Ready for final review.

**Pull request merged**